src/Dialogue/Confirm.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace PhpSchool\CliMenu\Dialogue;
+
+/**
+ * @author Aydin Hassan <aydin@hotmail.co.uk>
+ */
+class Confirm extends Dialogue
+{
+
+    /**
+     * Display confirmation with a button with the given text
+     *
+     * @param string $confirmText
+     */
+    public function display($confirmText = 'OK')
+    {
+        $this->assertMenuOpen();
+
+        $this->terminal->moveCursorToRow($this->y);
+
+        $promptWidth = mb_strlen($this->text) + 4;
+
+        $this->emptyRow();
+
+        $this->write(sprintf(
+            "%s  %s  %s\n",
+            $this->style->getUnselectedSetCode(),
+            $this->text,
+            $this->style->getUnselectedUnsetCode()
+        ));
+
+        $this->emptyRow();
+
+        $confirmText = sprintf(' < %s > ', $confirmText);
+        $leftFill = ($promptWidth / 2) - (mb_strlen($confirmText) / 2);
+
+        $this->write(sprintf(
+            '%s%s%s',
+            $this->style->getUnselectedSetCode(),
+            str_repeat(' ', $leftFill),
+            $this->style->getUnselectedSetCode()
+        ));
+
+        $this->write(
```

Comment:
yeah no worries sounds good to me. it would complicate centering height etc too so a nice little challenge in the future